ruma-events/src/policy/rule/room.rs

```diff
@@ -0,0 +1,91 @@
+//! Types for the *m.policy.rule.room* event.
+
+use ruma_events_macros::StateEventContent;
+
+use serde::{Deserialize, Serialize};
+
+use crate::{policy::rule::PolicyRuleEventContent, StateEvent};
+
+/// This event type is used to apply rules to room entities.
+pub type RoomEvent = StateEvent<RoomEventContent>;
+
+/// The payload for `RoomEvent`.
+#[derive(Clone, Debug, Deserialize, Serialize, StateEventContent)]
+#[ruma_event(type = "m.policy.rule.room")]
+pub struct RoomEventContent(PolicyRuleEventContent);
+
+#[cfg(test)]
+mod tests {
+    use super::{RoomEvent, RoomEventContent};
+
+    use crate::{
+        policy::rule::{PolicyRuleEventContent, Recommendation},
+        Unsigned,
+    };
+
+    use ruma_common::Raw;
+    use ruma_identifiers::{event_id, room_id, user_id};
+
+    use serde_json::{from_value as from_json_value, json, to_value as to_json_value};
+
+    use std::time::{Duration, UNIX_EPOCH};
+
+    #[test]
+    fn serialization() {
+        let room_event = RoomEvent {
+            event_id: event_id!("$143273582443PhrSn:example.org"),
+            sender: user_id!("@example:example.org"),
+            origin_server_ts: UNIX_EPOCH + Duration::from_millis(1_432_735_824_653),
+            room_id: room_id!("!jEsUZKDJdhlrceRyVU:example.org"),
+            state_key: "rule:#*:example.org".to_string(),
```

Comment:
I have a slight distaste for using `.to_string()` for `&str` -> `String` conversion and have previously updated all uses of that in Ruma to use `.into()`, could you change these too (mostly for consistency)?

Reply:
Changed it for my PR but some remains in the crate.